x-pack/plugins/observability/public/pages/alert_details/components/page_title.tsx

```diff
@@ -73,9 +73,9 @@ export function PageTitle({ alert, dataTestSubj }: PageTitleProps) {
         <EuiFlexItem grow={false}>
           <AlertLifecycleStatusBadge
             alertStatus={
-              alert.fields[ALERT_STATUS] === 'active'
+              status === 'active'
```

Comment:
Can we move this logic inside of `AlertLifecycleStatusBadge`? I remember I mentioned it on a PR that was already merged (a bit late I guess :D)

Like adjusting this function: https://github.com/elastic/kibana/blob/main/packages/kbn-alerts-ui-shared/src/alert_lifecycle_status_badge/index.tsx#L56,L83

Reply:
I checked the logic again, and it seems we don't need to check alert status here. I refactored the code in https://github.com/elastic/kibana/pull/174153/commits/772c2ae116ee8e0a2e7c36f47b5197407a9bdc1d.